src/olympia/lib/settings_base.py

```diff
@@ -1057,6 +1059,27 @@ def JINJA_CONFIG():
     'olympia.lib.es.management.commands.reindex',
 )
 
+CELERY_QUEUES = (
```

Comment:
Can you add a test that all queues used in `CELERY_ROUTES` + `CELERY_DEFAULT_QUEUE` are defined in `CELERY_QUEUES` ?